crates/iota/src/keytool.rs

```diff
@@ -589,8 +589,15 @@ impl KeyToolCommand {
             } => match IotaKeyPair::decode(&input_string) {
                 Ok(ikp) => {
                     info!("Importing Bech32 encoded private key to keystore");
-                    let key = Key::from(&ikp);
-                    keystore.add_key(alias, ikp)?;
+                    let mut key = Key::from(&ikp);
+                    keystore.add_key(alias.clone(), ikp)?;
+
```

Comment:
The alias match looked a bit strange at first glance, as we passed the alias to `.add_key()`
What do you think about adding such a comment?
 `// Use the provided alias or get the one set by .add_key() if None was provided`

Reply:
Do you think we still need a comment without the match?